package.json

```diff
@@ -7,7 +7,7 @@
     "Remoting",
     "REST"
   ],
-  "version": "2.34.0",
+  "version": "2.34.1",
```

Comment:
The version will automatically be updated when the release is cut. Incrementing it here means that v2.34.1 will be skipped unless it is specified when running `slt-release`.

Note that this is completely acceptable, I just wanted to make sure it was known.

Reply:
yes, good to know that the version is being handled automatically because it will be confusing if we merged some new changes and the version stays  the same as before.

Reply:
@smartmouse it's actually our convention in LoopBack to change the `version` field only when publishing a new release. Not a big deal here, but please don't update `version` field in the future.